src/iadmin.cpp

```diff
@@ -291,7 +293,7 @@ auto modify_replica(
             char* sub_error_name{};
             const char* error_name = rodsErrorName(status, &sub_error_name);
             std::cerr << "rcModDataObjMeta failed when modifying replica: [" << error_name << " (" << status << ")]" << std::endl;
-            printErrorStack(Conn->rError);
+            print_error_stack_to_file(stdout, Conn->rError);
```

Comment:
Why replace `printErrorStack` here?

This feels like a change in behavior. I imagine changing the output stream to stdout means the tests have to change.

_This question also applies to other locations where `printErrorStack` has been replaced._

Reply:
Yes, `printErrorStack` must do what it did before. People may have built clients that rely on that behavior.

Has that change to `printErrorStack` been merged?

Reply:
No it has not been merged. The change to `printErrorStack` is in the companion PR here https://github.com/irods/irods/pull/7149.

Reply:
Good. So, we restore the original behavior and this PR shrinks.